app/assets/javascripts/discourse/app/models/topic.js

```diff
@@ -795,18 +795,21 @@ Topic.reopenClass({
     return promise;
   },
 
-  bulkOperation(topics, operation) {
+  bulkOperation(topics, operation, tracked) {
+    const data = {
+      topic_ids: topics.map((t) => t.get("id")),
+      operation,
+      tracked,
+    };
+
     return ajax("/topics/bulk", {
       type: "PUT",
-      data: {
-        topic_ids: topics.map((t) => t.get("id")),
-        operation,
-      },
+      data,
     });
   },
 
-  bulkOperationByFilter(filter, operation, options) {
-    let data = { filter, operation };
+  bulkOperationByFilter(filter, operation, options, tracked) {
```

Comment:
I think that's slightly annoying to have tracked as an arg after options, it's generally the last arg if possible, can't we put tracked in options ?

Reply:
Good call, I'll move it in.

Reply:
Actually @jjaffeux I don't think I want to move it into `options`. Both `bulkOperation` and `bulkOperationByFilter` need to know about "tracked", and `bulkOperation` does not have `options` as a parameter. The way these functions are called also makes me want to keep it as a separate parameter.